frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -47,13 +47,14 @@ export const ClusterServiceVersionHeader: React.SFC = () => <ListHeader>
   <ColHead className="col-lg-3 col-md-3 hidden-sm hidden-xs">Provided APIs</ColHead>
 </ListHeader>;
 
-const menuActions = [Kebab.factory.Edit, Kebab.factory.Delete];
+const menuActions = [Kebab.factory.Edit];
 
 export const ClusterServiceVersionRow = withFallback<ClusterServiceVersionRowProps>(({obj}) => {
   const route = `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${obj.metadata.name}`;
 
+  const statusString = _.get(obj, 'status.reason', ClusterServiceVersionPhase.CSVPhaseUnknown);
   const installStatus = obj.status && obj.status.phase !== ClusterServiceVersionPhase.CSVPhaseFailed
-    ? <span>{_.get(obj, 'status.reason', ClusterServiceVersionPhase.CSVPhaseUnknown)}</span>
+    ? <span>{(statusString === 'Copied' || statusString === 'InstallSucceeded') && <i aria-hidden="true" className="pficon pficon-ok" />} {statusString}</span>
```

Comment:
This should use the `co-icon-and-text` markup. (Same for the existing icon below.)